src/webview_provider/queryResultPanel.ts

```diff
@@ -185,6 +187,61 @@ export class QueryResultPanel implements WebviewViewProvider {
                 );
             }
             break;
+          case InboundCommand.ShareQueryResult:
+            window.withProgress(
+              {
+                location: ProgressLocation.Notification,
+                title: "Uploading data to Altimate backend",
+                cancellable: false,
+              },
+              async () => {
+                try {
+                  const result = await this.altimate.shareQueryResult(message);
+                  if (result?.share_url) {
+                    await env.clipboard.writeText(result?.share_url);
+                    window.showInformationMessage("URL is copied to clipboard");
+                  } else {
+                    window.showErrorMessage(
+                      "Unable to upload data to Altimate backend.",
+                    );
+                  }
+                } catch (error) {
+                  window.showErrorMessage(
+                    `Unable to upload data to Altimate backend:${error}`,
+                  );
+                }
+              },
+            );
+            break;
+          case InboundCommand.InviteUser:
+            const { role, email } = message;
+            const company = workspace
+              .getConfiguration("dbt")
+              .get<string>("altimateInstanceName");
+            if (!company) {
+              window.showErrorMessage(
+                "Please instance name and API key for inviting user.",
```

Comment:
```suggestion
                "Please add instance name and API key for inviting user.",
```

Reply:
why dont you use `handlePreviewFeatures` check instead of checking just instance name?